packages/web-components/fast-foundation/src/accordion-item/accordion-item.spec.ts

```diff
@@ -1,15 +1,15 @@
 import { expect } from "chai";
-import { AccordionItem, accordionItemTemplate as template } from "./index.js";
-import { fixture } from "../testing/fixture.js";
+import { FASTAccordionItem, accordionItemTemplate } from "./index.js";
+import { fixture, uniqueElementName } from "../testing/fixture.js";
 
-
-const FASTAccordionItem = AccordionItem.compose({
-    baseName: "accordion-item",
-    template
-})
+const name = uniqueElementName();
```

Comment:
How might using a randomly-generated element name affect the reproducibility and predictability of test results? Is there an added value to using `fast-unique-o32lyt` over a known static value in tests?

Reply:
I was trying not to make any assumptions about the test runtime environment and how it might sandbox the tests. So, I wanted to ensure that individual tests could declare their own components without the possibility of name collision. Additionally, since we do have people using these with all sorts of different names, I thought it would help validate that use case more broadly.